### PR TITLE
Regenerate pydoc topics file

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

**Comment by pablogsal** [comment]
For some reason, the newlines looks wrong....investigating...

File: Lib/pydoc_data/topics.py [hunk 1]
```diff
@@ -2558,22 +2558,25 @@
              '(see\n'
              'section The standard type hierarchy):\n'
              '\n'
-             '   funcdef                 ::= [decorators] "def" funcname "(" '
+             '   funcdef                   ::= [decorators] "def" funcname "(" '
              '[parameter_list] ")"\n'
              '               ["->" expression] ":" suite\n'
-             '   decorators              ::= decorator+\n'
-             '   decorator               ::= "@" dotted_name ["(" '
+             '   decorators                ::= decorator+\n'
```
Comment (eamanu) [inline Lib/pydoc_data/topics.py:2564]: Why white-spaces is added?

Comment (pablogsal) [reply Lib/pydoc_data/topics.py:2564]: This file is automatically generated

**Comment by Carreau** [comment]
Out of curiosity how are these regenerated, and where can I find this information ?

**Comment by pablogsal** [comment]
@Carreau `make pydoc-topics` in the Docs folder, but usually is done by the release manager before releases (I just learned that fact looking at the git history for that file).

**Pull request merged**